jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory/generic-files.md

```diff
@@ -266,7 +266,7 @@ jf rt cp "source-frog-repo/rabbit/*" target-frog-repo/rabbit/ --props=Version=1.
 
 #### Example 4
 
-Copy all artifacts located under /rabbit in the **source-frog-repo** repository to an absulote path.
+Copy all artifacts located under /rabbit in the **source-frog-repo** repository into the same path in the target-frog-repo repository without maintaining the original subdirectory hierarchy.
```

Comment:
target-frog-repo --> **target-frog-repo**
(Notice the markdown ** wrapping)

The same goes for the mv command example.